### PR TITLE
Opencv practice

### PR DESCRIPTION
**Pull request by VincentHawks**

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```
Comment (dkurt) [inline modules/0_git/test/main.cpp]: Please revert this change (next just start every new task from master branch):
```
git checkout master
git checkout -b new_task
```

**Comment by VincentHawks** [comment]
It passes the test locally, no idea what's wrong

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -10,5 +11,120 @@ unsigned countCoins(const Mat& img) {
...
+		{
+			value = markers.at<int>(Point(x, y));
+			// This can be replaced with a single if, but i'm not gonna bother
+			switch (value)
+			{
+			case 1:
+				//gray.at<uint8_t>(Point(x, y)) = 0;
+				break;
+			case -1:
+				//gray.at<uint8_t>(Point(x, y)) = 255;
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp:82]: Why it is commented?

Comment (VincentHawks) [reply modules/1_opencv/src/opencv_coins.cpp:82]: It's just for the visualisation

File: modules/1_opencv/src/opencv_coins.cpp [hunk 3]
```diff
@@ -10,5 +11,120 @@ unsigned countCoins(const Mat& img) {
...
+	Mat sure_bg;
+	dilate(thresh, sure_bg, Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
+	sure_bg.convertTo(sure_bg, CV_8U);
+
+	Mat dist_transform;
+	distanceTransform(thresh, dist_transform, DIST_L2, 5);
+	dist_transform.convertTo(dist_transform, CV_8U);
+
+	// Find the max in the dist_transform
+	uint8_t max;
```
Comment (dkurt) [inline modules/1_opencv/src/opencv_coins.cpp]: uninitialized variable

File: modules/0_git/test/main.cpp (hunk 1 above)
Comment (dkurt) [reply modules/0_git/test/main.cpp]: please not forget to fix it as well

Comment (VincentHawks) [reply modules/0_git/test/main.cpp]: just a moment

**Comment by VincentHawks** [comment]
This is supposed to fail, given that i've de-solved the task in module 0, right?
